spec/resources.yml

```diff
@@ -25,26 +25,108 @@ components:
 
-
         If the parameter `count` is `true`, returns only the number of items in the list.
 
-
         ##### Text search
 
-
         If the `search` parameter is provided, narrows results to those pertaining to the search query. Search works across resource IDs and the values of annotations. It weighs results so that those with matching id or a matching value of an annotation called `name` appear first, then those with another matching annotation value, and finally those with a matching  `kind`."
         operationId: "getResources"
         parameters:
+        - name: "account"
+          in: "query"
+          description: "Organization account name"
+          example: myorg
+          schema:
+            $ref: 'openapi.yml#/components/schemas/AccountName'
+        - name: "kind"
+          in: "query"
+          description: "Type of resource"
+          example: user
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Kind'
+        - name: "search"
+          in: "query"
+          description: "Filter resources based on this value by name"
+          example: "db"
+          schema:
+            $ref: '#/components/schemas/SearchFilter'
+        - name: "offset"
+          in: "query"
+          description: "When listing resources, start at this item number."
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Offset'
+        - name: "limit"
+          in: "query"
+          description: "When listing resources, return up to this many results."
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Limit'
+        - name: "count"
+          in: "query"
+          description: "When listing resources, if `true`, return only the count of the results."
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Count'
+        - name: "role"
+          in: "query"
+          description: "Retrieves the resources list for a different role if the authenticated role has access"
+          example: myorg:host:host1
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Role'
+        - name: "acting_as"
+          in: "query"
+          description: "Retrieves the resources list for a different role if the authenticated role has access"
+          example: myorg:host:host1
+          schema:
+            $ref: 'openapi.yml#/components/schemas/Role'
+
+        responses:
+          "200":
+            description: "The response body contains the list of matching resources"
+            content:
+              application/json:
+                schema:
+                  type: object
+          "401":
+            $ref: 'openapi.yml#/components/responses/UnauthorizedError'
+          "403":
+            $ref: 'openapi.yml#/components/responses/InadequatePrivileges'
+          "422":
+            $ref: 'openapi.yml#/components/responses/UnprocessableEntity'
+          "500":
```

Comment:
```suggestion
```
I wasn't able to get a 500 response from the contract test when testing, and it is not listed as a possible response code on the docs

Reply:
I initially added these responses in my API contract test PR, as the test did produce some server errors. I removed them and the contract test passes, so I guess it was unrelated.